projects/core/src/configurator/commons/store/reducers/configurator-ui.reducer.spec.ts

```diff
@@ -81,4 +82,144 @@ describe('Configurator UI reducer', () => {
       expect(state.currentGroup).toEqual(null);
     });
   });
+
+  describe('Group Status reducers', () => {
+    it('Group Visited Reducer initial state', () => {
```

Comment:
it describes what the test **should** check, for instance **should prove the initial state** or something like this...